### ISSUE TITLE
Mailhog and Apache won't start after Windows 10 creators update

### ISSUE THREAD

**Issue by HollyIT** [crazy-max/neard#239]
After getting the final Windows 10 Creator update, which is being pushed out to all users starting next week, I can no longer start Apache or Mailhog (I only have Apache, Mailhog and MySQL enabled in Neard). They return a Status 7, service is paused.

MySQL starts without any problem, so I'm thinking it's a problem in nssm. Also I can start Apache from the command line and it works fine (haven't tried Mailhog, but I suspect it would be the same thing). I can also install Apache as a service using the "-k install" and all works well. 

I'm running the latest version (1.0.22) on Windows 10 Pro 64bit. 

Here's a link to the logs and tmp files:
https://www.sendspace.com/file/gfxvp3

**Comment by crazy-max** [comment]
Hi, thanks for your feedback
I will investigate on jour issue

**Comment by crazy-max** [comment]
As it is stated on the official website, there is a glitch linked to services requiring console window on Windows 10 Creators Update Redstone 2 : https://nssm.cc/download

We have to add an additionnal parameter as workaround :

```
nssm set <service> AppNoConsole=1
```

**Comment by HollyIT** [comment]
Ahh. I checked their site yesterday and that wasn't there. I just did a quick hack class.nssm.php:

`    $exec = $this->exec('set ' . $this->getName() . ' AppNoConsole "1"');
    if ($exec === FALSE) {
      return FALSE;
    }
`
Note it's not AppNoConsole=1, but rather nssm set <service> AppNoConsole "1". I tried both in the class and from the CLI using = and without quotes and none of those would take, so it has to be encapsulated in quotes to take.

And after figuring that part out, both Apache and Mailhog are starting and working.

Thanks alot!
